pom.xml

```diff
@@ -478,4 +478,4 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
-</project>
\ No newline at end of file
+</project>
```

Comment:
is this expected to be changed?

Reply:
that was not, not sure why it got changed. however I don't see any harm in removing the whitespace at the end. leaving as is